src/test/java/pl/com/bottega/ecommerce/sales/domain/invoicing/BookKeeperTest.java

```diff
@@ -1,20 +1,89 @@
 package pl.com.bottega.ecommerce.sales.domain.invoicing;
 
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
 
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.com.bottega.ecommerce.canonicalmodel.publishedlanguage.ClientData;
+import pl.com.bottega.ecommerce.canonicalmodel.publishedlanguage.Id;
+import pl.com.bottega.ecommerce.sales.domain.productscatalog.Product;
+import pl.com.bottega.ecommerce.sales.domain.productscatalog.ProductType;
+import pl.com.bottega.ecommerce.sharedkernel.Money;
 
+
+@ExtendWith(MockitoExtension.class)
 class BookKeeperTest {
+    static ClientData clientData;
+    static BookKeeper bookKeeper;
+    static Product product;
+    static InvoiceRequest invoiceRequest;
+    static RequestItem requestItem;
+
+    @Mock
+    static TaxPolicy taxPolicy;
+
+    @BeforeAll
+    static void setUp() {
+        bookKeeper = new BookKeeper(new InvoiceFactory());
+        clientData = new ClientData(Id.generate(), "Tomasz");
```

Comment:
to nie jest dobre podejście - dane statyczne - czyli takie same dla wszystkich testów. Poza tym powoduje to, że jakiś pośredni stan może wpłynąć na kolejne testy (statyczne pola nie są reinicjalizowane przed każdym testem).